plugins/telemetry-otel/config/telemetry-otel/log4j2.properties

```diff
@@ -41,7 +41,7 @@ appender.metrics.strategy.type = DefaultRolloverStrategy
 appender.metrics.strategy.max = 4
 
 
-logger.metrics_exporter.name = io.opentelemetry.exporter.logging.LoggingMetricExporter
+logger.metrics_exporter.name = io.opentelemetry.exporter.logging.OtlpJsonLoggingMetricExporter
```

Comment:
AFAIK, the OpenSearch logs are written as unstructured (*.log) and structured (*.json), if we want to support structured logs for OTEL exporters, we have to update that.